front/src/components/mmt-locale-switcher/mmt-locale-switcher.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="locale-switcher">
+  <v-select :items="locales"
+            v-on:change="switchLocale"
+            :label="$t('footer.language')"
+            v-model="this.$root.$i18n.locale">
+    <template slot="selection" slot-scope="data">
+      {{ $t(data.item.text) }}
+    </template>
+    <template slot="item" slot-scope="data">
+      {{ $t(data.item.text) }}
+    </template>
```

Comment:
why did you choose slot? it has simpler way - props

Reply:
I removed the use of slots. Replaced by computed property. Display text in v-select dynamically changes depending on the selected language.